Github-Org-Management-Policy.md

```diff
@@ -0,0 +1,50 @@
+# Node.js GitHub Organization Management Policy
+
+The Node.js Foundation Github Organization (https://github.com/nodejs) is
+provided as a development resource by the Node.js Foundation under the
+operational direction of the Node.js Technical Steering Committee (TSC).
+
+## Organization Roles
+
+Individual users within a GitHub organization may be granted one of three
+distinct roles: Member, Owner, or Billing manager. The permissions granted
+to each role are documented in the GitHub documentated [Permission levels for
+an organization](https://help.github.com/articles/permission-levels-for-an-organization/).
+
+### Billing manager
+
+All Billing managers for the Node.js Foundation GitHub Organization shall
+be determined by the Node.js Foundation Board of Directors with notification
+given to the TSC.
+
+### Owners
+
+All voting members of the Node.js TSC shall have Owner permissions within the
+Node.js Foundation GitHub Organization. Additional users may be granted Owner
+permissions at the discretion of the TSC through normal TSC motion and simple
+majority vote.
```

Comment:
Well, the moderation policy and other bits existed before the CommComm did so it was impossible to take those into consideration really. The intent prior to the formation of the CommComm was for the Github organization to be directed operationally by the TSC in nearly every aspect. This notion that it would be co-managed by both is new and is not something that I think has been fully digested. We need to take some time to understand what exactly it means and what the ramifications are. I'm sure there are also concerns. For instance, what happens if the CommComm adopts a different moderation policy than the TSC, which one applies in any given instance if the GitHub organization is being managed by both? It can be a tricky exercise.

One suggestion that has been made in a couple of conversations I've had is that it may be easiest for the Community Committee to actually maintain it's own separate GitHub organization, much the same way that the Express project and libuv maintain their own GitHub organizations. There's absolutely nothing that requires the CommComm to use the existing github.com/nodejs organization. It may be the easiest and least controversial approach to take.